buildSrc/src/main/rocker/com/uber/okbuck/template/android/RobolectricRule.rocker.raw

```diff
@@ -0,0 +1,50 @@
+@import java.util.Collection
```

Comment:
Lets rename to `AndroidTestRule` since in the future, there will be such rules that dont use robolectric as well

Reply:
I'd call it `AndroidUnitTestRule`, as `AndroidTest` can be cause confusion, as in gradle it means instrumentation tests

Reply:
AGP source code is different from okbuck source code. To keep with the conventions of the other classes, lets use `AndroidTest`